utils/maintenence.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
```

Comment:
I'm sure these could be cleaned up in the future, but that is a feature. I do not think it is rare to want to do them all, as doing them all is required on each release per CEP3.